src/utils/notify_plugins/notify_email.py

```diff
@@ -15,8 +18,13 @@ def send_email(subject, to, html, journal, request, bcc=None, cc=None, attachmen
     else:
         from_email = request.press.main_contact
 
-    if not type(to) in [list, tuple]:
-        to = [to]
+    if isinstance(to, str):
+        if DUMMY_EMAIL_DOMAIN in to:
+            to = []
```

Comment:
I suspect Mailgun will give a 400 if `to` is empty.

Reply:
I'll handle that  case then

Reply:
Actually, seems like Django has thought of this for us :)
https://github.com/django/django/blob/58f02c498b659a906e9c30d946bd89bedc4717e5/django/core/mail/message.py#L336